gc/gc-base/src/main/java/org/projectnessie/gc/base/GCUtil.java

```diff
@@ -55,6 +57,13 @@ public static Reference deserializeReference(String reference) {
     }
   }
 
+  public static Instant getInstantFromMicros(Long microsSinceEpoch) {
```

Comment:
since we are making it public, we need unit tests for it

Reply:
Added